core/src/main/java/edu/wpi/grip/core/http/GripServer.java

```diff
@@ -100,6 +106,32 @@
    */
   public static final String DATA_PATH = ROOT_PATH + "/data";
 
+  /**
+   * The default port the server should run on.
+   */
+  public static final int DEFAULT_PORT = 8080;
```

Comment:
I think it's okay because we're hosting an HTTP server on the port. If it was something proprietary like a connection to a central server, it would be on a different port